chainbase/src/main/java/org/tron/core/db/TronStoreWithRevoking.java

```diff
@@ -56,10 +54,7 @@
   protected TronStoreWithRevoking(String dbName) {
     int dbVersion = CommonParameter.getInstance().getStorage().getDbVersion();
     String dbEngine = CommonParameter.getInstance().getStorage().getDbEngine();
-    if (dbVersion == 1) {
-      this.revokingDB = new RevokingDBWithCachingOldValue(dbName,
-          getOptionsByDbNameForLevelDB(dbName));
-    } else if (dbVersion == 2) {
+    if (dbVersion == 2) {
```

Comment:
should reserve the check for v1, and throw an exception when people try to use v1 to start the node?

Reply:
It will throw a runtime exception when dbVersion set to 1 as the else logic will take effect.